yandextank/api/apiworker.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 """ Provides class to run TankCore from python """
-from yandextank.core import tankcore
 import logging
-from pkg_resources import resource_filename
 import os
 import sys
 import time
 import traceback
+import fnmatch
+from pkg_resources import resource_filename
+
+from ..core import tankcore
```

Comment:
Relative imports are not discouraged. 
I saw PEP 328 and I think it's our use case where we should use relative imports because modules (or parts of tank) is useless outside of our package. Such usage of imports allows us to use multiple yandextank installations on one machine without virtualenv and to be sure that we use appropriate plugins/parts of tank. At last, we have some plugins where we use relative imports, some plugins where we use absolute imports. My commit makes everything to one style. It doesn't broke 'readability', because almost all functionality of tank is in plugins and it has desirable structure.

Absolute imports for system packages. Relative - for parts of tank. If you see where I'm wrong, tell me. 

+summon @direvius @nnugumanov